spec/components/modal/Modal.spec.tsx

```diff
@@ -31,4 +38,26 @@ describe('Modal', () => {
     expect(wrapper.find('ModalHeader').length).toBe(0);
     expect(wrapper.find('ModalBody').length).toBe(0);
   })
+  it('should close the Modal on "Esc"', async () => {
+    let show = true;
+    const handleClose = () => { show = false }
+    render(<Modal size={'small'} show={show} onClose={handleClose}>
+      <ModalHeader>Hello, World</ModalHeader>
+      <ModalBody>Some text</ModalBody>
+    </Modal>);
+
+    // Modal should be displayed
+    const title = screen.getByText('Hello, World');
+    expect(title).toBeTruthy();
+
+    const eventMock = {
+      key: Keys.ESC,
+    };
+
+    // 'ESC' key in the Modal
+    fireEvent.keyUp(title, eventMock);
+
+    // Modal should be closed
+    expect(show).toBeFalsy();
```

Comment:
To have a comparison before/after in your test, I suggest to add either : 
- expect(title).toBeFalsy(); (line 62)
- or expect(show).toBeTruthy(); (line 52)

Reply:
Ok.Done